### DISCUSSION TITLE
Dropped support for .NET Standard 2.0

### DISCUSSION THREAD

**Discussion by FantasyTeddy** [Marfusios/websocket-client#140] [Q&A]
Is there a particular reason, why the library is not targeting `netstandard2.0` anymore?

This removes an entire generation of projects (everything still on .NET Framework) from being able to update to this version.
(In particular the old dependency to `LibLog` in the older versions is unfortunate.)

**Comment by spudwebb** [comment]
I use this library too for a .NET framework project and can' t update it anymore because netstandard2.0 has been dropped  :disappointed:

**Comment by RevZero** [comment]
I was just looking to update this for a Framework 4.7.2 project and wondered about this as well. At a glance, I couldn't make out any changed that would depend on Standard 2.1.

**Reply by Marfusios** [reply]
Maybe we can bring it back, if I remember correctly, the main reason was the new C# and .NET library features that are not available under netstandard. 
I wanted to avoid having pragma #ifs all over the codebase
